src/containers/my-cooperations/cooperation-details/CooperationDetails.tsx

```diff
@@ -132,6 +152,17 @@ const CooperationDetails = () => {
     return cooperationContent
   }
 
+  const isCooperationClosingRequestSend =
+    response.needAction === userRole &&
+    response.status === StatusEnum.RequestToClose
+
+  const acceptClosingProccess = !isClosed && (
```

Comment:
```suggestion
  const acceptClosingProcess = !isClosed && (
```
A little typo here. Install [this](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) vs code extention and it will help you to notice typos